add project files Sprint_04

SaveliiSidorkin opened the pull request:
FragmentMainPage - локаторы и методы главной страницы https://qa-scooter.praktikum-services.ru/
FragmentFirstOrderPage - локаторы и методs первой страницы заказа (поиск элемента, нажатие, заполнение)
FragmentSecondOrderPage - локаторы и методы второй страницы заказа (поиск элемента, нажатие, заполнение)
BasePage - хранится WebDriver и WebDraiverWait
MainPage - хранится URL
BaseTest - открытие, закрытие, выбор другого браузера
CheckTextInAnswersTest - тест на проверку содержания блока "Вопросы о важном"
OrderSamokatTest - тест на проверку заказа самоката

Commit:
add project files Sprint_04

egorvas commented on src/test/java/ru/praktikum_services/qa_scooter/OrderSamokatTest.java:
```diff
@@ -0,0 +1,66 @@
+    }
+
+    @Test
+    public void CheckOrder() {
```
**Нужно исправить:** функции java только с большой буквы

egorvas commented on src/test/java/ru/praktikum_services/qa_scooter/OrderSamokatTest.java:
```diff
@@ -0,0 +1,66 @@
+        mainPage.clickFirstOrderButton();
+        mainPage.backToMainPage();
+        mainPage.scrollToSecondOrderButton();
+        mainPage.clickSecondOrderButton();
```
**Нужно исправить:** по условиям задания нужно проверить заказ полностью с обоими кнопками. А у тебя получается один заказ со второй кнопкой. Реконмедую тебе кнопку заказа вынести в параметр.

Commit:
Add fix:
1.Use camelCase in method in OrderSamokatTest.java
2.Changed logic of clicking on the order button in OrderSamokatTest.java and FragmentMainPage

SaveliiSidorkin commented:
1. Исправил название метода в OrderSamokatTest.java
2.Вынес клик по кнопке заказа в параметр
